### DISCUSSION TITLE
Remove support for Node < 14

### DISCUSSION THREAD

**Discussion by j0k3r** [serverless-heaven/serverless-webpack#1562] [Team Posts]
AWS Lambda doesn't support Node < 14 anymore.
https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

They will also deprecate Node 14 in November 2023.

I suggest to ditch Node 10 & Node 12 support in the next release (ie: 5.13.0). What do you think?

**Comment by vicary** [comment]
Simplifying the compatibility matrix makes everything easier, also lowers the barriers to new contributors.

Technically it is breaking, but for Lambda to continue using Node 12 it has to be deployed via custom image. Somewhat niche TBH, if we all agree it's tolerable then we can go for minor.

**Comment by j0k3r** [comment]
Released https://github.com/serverless-heaven/serverless-webpack/releases/tag/v5.13.0
